ztp/policygenerator/policyGen/policyBuilder.go

```diff
@@ -232,27 +248,36 @@ func (pbuilder *PolicyBuilder) splitYamls(yamls []byte) ([][]byte, error) {
 	return resources, nil
 }
 
-func (pbuilder *PolicyBuilder) readAcmPolicyTemplate() (utils.AcmPolicy, error) {
+var (
```

Comment:
Lets consider that for a future cleanup. I think there may be advantages to leaving it as a resource like this. For example, if the policy is missing some fixed/static field it could be added to the container as a patch (temporary fix). If the string is compiled-in we have to roll a new binary.